docs/src/atomistic/engines/lammps.rst

```diff
@@ -0,0 +1,134 @@
+.. _engine-lammps:
+
+LAMMPS
+======
+
+.. list-table::
+   :header-rows: 1
+
+   * - Official website
+     - How is metatensor supported?
+   * - https://lammps.org
+     - In a separate fork
+
+
+Supported model outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Only the :ref:`energy <energy-output>` output is supported in LAMMPS, as a
+custom ``pair_style``. This allows running molecular dynamics simulations with
+interatomic potentials in metatensor format; distributing the simulation over
+multiple nodes and potentially multiple GPUs.
+
+How to install the code
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The code is available in a custom fork of LAMMPS,
+https://github.com/metatensor/lammps. We recommend you use `LAMMPS' CMake build
+system`_ to configure the build.
+
+To build metatensor-enabled LAMMPS, you'll need to provide the C++ version of
+``libtorch``, either by installing PyTorch with a Python package manager
+(``pip`` or ``conda``), or by downloading the right prebuilt version of the code
+from https://pytorch.org/get-started/locally/. To build metatensor itself, you
+will also need to install a Rust compiler and ``cargo`` installed, either with
+`rustup`_ or the package manager of your operating system.
+
+First, you should run the following code in a bash (or bash-compatible) shell to
+get the code on your system and to teach CMake where to find ``libtorch``:
+
+.. code-block:: bash
+
+    # point this to the path where you extracted the C++ libtorch
+    TORCH_PREFIX=<path/to/torch/installation>
+    # if you used Python to install torch, you can do this:
+    TORCH_PREFIX=$(python -c "import torch; print(torch.utils.cmake_prefix_path)")
+
+    # patch a bug from torch's MKL detection
+    git clone https://github.com/metatensor/lammps lammps-metatensor
+    cd lammps-metatensor
+    ./src/ML-METATENSOR/patch-torch.sh "$TORCH_PREFIX"
+
+After what you can configure the build and compile the code:
+
+.. code-block:: bash
+
+    mkdir build && cd build
+
+    # you can add more options here to enable other packages.
+    cmake -DPKG_ML-METATENSOR=ON -DCMAKE_PREFIX_PATH="$TORCH_PREFIX" ../cmake
```

Comment:
should we add `LAMMPS_INSTALL_RPATH=ON`?

Reply:
hmm, I would rather put it directly in the code so you don't have to find the docs to use it.

Reply:
Sure!

Reply:
actually I don't think we can add it in the code, so let's put it here as a next best thing.